distributed/deploy/cluster.py

```diff
@@ -114,6 +114,15 @@ def loop(self, value: IOLoop) -> None:
             raise ValueError("expected an IOLoop, got None")
         self.__loop = value
 
+    @property
+    def called_from_running_loop(self):
+        try:
+            return (
+                getattr(self.loop, "asyncio_loop", None) is asyncio.get_running_loop()
+            )
+        except RuntimeError:
+            return self.asynchronous
```

Comment:
this should be the value passed in `def __init__(self, *, ..., asynchronous=False)` not the computed attribute

Reply:
Ok I've updated it to store that in a private attribute to avoid folks using it instead of the attribute..